lib/app_constants.dart

```diff
@@ -11,13 +11,17 @@ class RoutePaths {
   static const String Notifications = 'notifications';
   static const String Profile = 'profile';
   static const String CardsView = 'profile/cards_view';
-  static const String NotificationsFilter = 'notifications/filter';
+  static const String NotificationsSettingsView =
+      'notifications/notifications_settings';
 
   static const String NewsViewAll = 'news/newslist';
   static const String EventsViewAll = 'events/eventslist';
-  static const String EventsAll = 'events/events_view_all';
+  static const String EventsAll = 'triton_media/triton_media_view_all';
```

Comment:
Re-add line 18
static const String EventsAll = 'events/events_view_all';

and rename line 19 EventsAll